tests/validation/test_highlights.py

```diff
@@ -0,0 +1,366 @@
+                        }
+                    ],
+                    "properties": [
+                        {
+                            "key": "stable_property",
+                            "stringValue": "I am stable",
+                            "timestamp": 1666813121048
+                        }
+                    ]
+                }
+            ]
+        }
+
+    def test_no_highlight(self):
+        response = TestObject()
+        response.is_success = True
+        response.text = json.dumps(pytest.JSON)
+        normal_collection = LongCollectionBundle(5, 25)
+        # Add ten identical collections
+        for collection in range(10):
+            collection_bundle = CollectionBundle(pytest.REQUEST, copy.deepcopy(response), pytest.DURATION,
+                                                 pytest.CONTAINER_STATISTICS)
+            collection_bundle.collection_number = collection
+            normal_collection.add(collection_bundle)
+
+        highlight = normal_collection.validate()
+        assert len(highlight.messages) == 0
+
+    def test_highlight_for_growing_unique_objects(self):
+        response = TestObject()
+        response.is_success = True
+        _json = copy.deepcopy(pytest.JSON)
+        growing_unique_objects_overtime = LongCollectionBundle(5, 25)
+        for collection in range(10):
+            response.text = json.dumps(_json)
+            collection_bundle = CollectionBundle(pytest.REQUEST, copy.deepcopy(response), pytest.DURATION,
+                                                 pytest.CONTAINER_STATISTICS)
+            collection_bundle.collection_number = collection
+            growing_unique_objects_overtime.add(collection_bundle)
+            _json["result"][0]["key"]["name"] = f"{collection}"
+
+        highlight = highlight_object_growth(growing_unique_objects_overtime.long_collection_statistics)
+        assert (ResultLevel.WARNING,
+                "Object of type HighlightsMP::Growing Object displayed growth of 9.65%") in highlight.messages
+
+    def test_highlight_for_growing_unique_objects_extreme_growth(self):
+        response = TestObject()
+        response.is_success = True
+        _json = copy.deepcopy(pytest.JSON)
+        growing_unique_objects_overtime = LongCollectionBundle(5, 25)
+        for collection in range(100):
+            response.text = json.dumps(_json)
+            collection_bundle = CollectionBundle(pytest.REQUEST, copy.deepcopy(response), pytest.DURATION,
+                                                 pytest.CONTAINER_STATISTICS)
+            collection_bundle.collection_number = collection
+            growing_unique_objects_overtime.add(collection_bundle)
+            if collection % 3 == 0:
+                _json["result"][0]["key"]["name"] = f"{collection}"
+
+        highlight = highlight_object_growth(growing_unique_objects_overtime.long_collection_statistics)
+        assert (ResultLevel.WARNING,
```

Comment:
Do warnings get displayed by default? If not I wonder if we should change that.

Reply:
I'm not sure ERROR is appropriate here, but I don't think they should be hiding in a log by default.